.github/workflows/rollback.yml

```diff
@@ -16,25 +16,39 @@ name: Release rollback
 on:
   workflow_dispatch:
     inputs:
-      rollback_version:
+      rollback-version:
         description: 'the version number to rollback to'
         required: true
+      skip-image-rollback:
```

Comment:
Maybe used default false for boolean type when the required is false though.

Reply:
Boolean types are false by default.